docs/standalone/audit_guidelines.md

```diff
@@ -0,0 +1,137 @@
+ - Coordinated Hydra Head V1 Specification
+ - Hydra plutus scripts (on-chain code)
+ - Hydra node chain layer code (off-chain code)
+
+As described in the following figure, the main entry points of a Hydra node are:
+* The API through which a client can connect to the node;
+* The network communication with other Hydra node peers;
+* The transactions posted on the Cardano ledger (accessed through a network connection).
+ 
+![artifacts.png](artifacts.png)
+
+A detailed description of each of the artifacts relevant for an audit can be found in the above section _Artifacts_.
+
+For its operations, the hydra-node process relies on a cardano-node process and client processes can connect to the hydra-node process through API.
+Generally, we recommand to run the Hydra and cardano node in a trusted environment so that any assessment performed during an audit should be done under the following assumptions:
+* Organizational procedures for managing the hydra and cardano signing keys are trusted;
+* The system running the hydra-node is trusted;
+* The cardano-node software is trusted as is the system running it;
+* Any client software connecting through the API is trusted as are the systems running them;
+* The communications between the cardano-node and the hydra-node are trusted;
+* The communications between the hydra-node and any client software connecting through the API are trusted;
+* The cardano-node is assumed to have a responsive communication line with the Cardano network such that the hydra-node can react to on-chain transactions in a timely manner. In particular, hydra-node is able to observe head closure and contest in a delay shorter than the contestation period fixed by the protocol.
+
+What is not explicitly trusted in the above list is deemed untrusted. In particular, the other hydra nodes participating in a head are not trusted.
+
+We would appreciate comments about any of the above. Especially if you can suggest less restricting assumptions under which the security properties of Hydra still hold. Or practical suggestions on how to implement these assumptions. We also appreciate recommendations about how to ensure these assumptions hold in a real environment.
+
+If you plan to consider a different operational environment were these assumptions would not be true and plan to share your discoveries with us, please be as explicit as possible about the corresponding assumptions.
+
+# Suggested Tasks
+
+Broadly speaking, an audit would probably want to ensure that the security properties proven in _Coordinated Hydra Head V1 Specification_ hold for the implementation, taking also into acount the main entry points of a `hydra-node` which are the node to node network communications, the API and the Cardano ledger.
```

Comment:
```suggestion
Broadly speaking, an audit would probably want to ensure that the security properties proven in _Hydra Head Specification_ hold for the implementation, taking also into acount the main entry points of a `hydra-node` which are the node to node network communications, the API and the Cardano ledger.
```

Reply:
And so on. I think we should call the artifact just "The Hydra Head Specification" and not much versioning here.